trailblazer/cli/core.py

```diff
@@ -69,31 +69,38 @@ def log_cmd(context, sampleinfo, sacct, quiet, config):
 
 
 @base.command()
-@click.option('-c', '--mip-config', type=click.Path(exists=True), help='MIP config')
+@click.option('-c', '--mip-config', 'pipeline_config', type=click.Path(exists=True),
```

Comment:
Shouldn't the option be named something different than mip-config?

Reply:
My intention was to change stuff without breaking compatibility and this seemed to have the potential to break something

Reply:
Alright!